component-library/src/app/components/component-preview/component-preview.component.scss

```diff
@@ -8,15 +8,23 @@
     align-items: center;
     justify-content: center;
     background-color: var(--surface2);
-    max-width: 362px;
     min-height: 170px;
 
     jds-button.copy {
       flex-basis: 100%;
       text-align: center;
       margin-top: -10px;
-      color: #2A4980;
+      color: #2a4980;
```

Comment:
Any chance we can replace this hex code with existing colour token?

Reply:
Sounds good.

Reply:
fixed.